operator/config/crd/kustomization.yaml

```diff
@@ -3,6 +3,7 @@
 # It should be run by config/default
 resources:
 - bases/operator.kyma-project.io_manifests.yaml
+- bases/sample_crd.yaml
```

Comment:
this will cause the main installation to install the sample crd (later on also in cli and pipelines), we should probably remove this

Reply:
this was for testing, I will remove this